src/UHC/Util/RLList.hs

```diff
@@ -23,7 +23,7 @@ import           Prelude hiding (length, init, null, concat)
 import qualified Prelude as P
 import           Data.Maybe
 import qualified Data.List as L
-import           Data.List hiding (concat, init, null, isPrefixOf, length, inits)
+import           Data.List hiding (concat, init, null, isPrefixOf, length, inits, singleton)
```

Comment:
doesn't this conflict with the singleton defined in the module?

Reply:
This is a `hiding` list, so I had to add it to avoid conflicts.

Reply:
ack, will put my glasses on...